docs/content/en/docs/concepts/architecture/deployment-flow.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Flow of deployment
+linktitle: Flow of deployment
+description: Understand the execution flow of a deployment
+weight: 35
+---
+
+The Keptn Lifecycle Toolkit (KLT) deploys a
+[Kubernetes Workload](https://kubernetes.io/docs/concepts/workloads/)
+by passing through a well-defined execution flow.
+
+The execution flow goes through six main phases:
+
+* Pre-deployments-tasks
+* Pre-deployment-evaluation
+* Deployment
+* Post-deployment-tasks
+* Post-deployment-evaluation
+* Completed
+
+Within each phase,
+the order of execution is determined
+by the order in which evaluations and tasks are listed in the
+[KeptnApp](../../yaml-crd-ref/app/)
+resource.
+
+## Kubernetes events
+
+A [Kubernetes Event](https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/event-v1/)
+is emitted at each phase
```

Comment:
There can potentially be multiple events that are sent during a phase, so it does not necessarily always have to be one:

```suggestion
[Kubernetes Events](https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/event-v1/)
are emitted at each phase
```